docs/security/security-trusted-workspace-access.md

```diff
@@ -100,7 +100,8 @@ The following sections show you how to use these methods.
 * Configure a [resource instance rule](#resource-instance-rule) for the storage account.
 
 > [!NOTE]
-> Preexisting shortcuts in a workspace that meets the prerequisites will automatically start to support trusted service access.
+>- Preexisting shortcuts in a workspace that meets the prerequisites will automatically start to support trusted service access.
+>- You must use the dfs url ID for the storage account. Ex: https://StorageAccountName.dfs.core.windows.net
```

Comment:
Hi @abudhu , can you wrap the example URL in back tics, and change Ex: to For example. i.e.
For example, `https://StorageAccountName.dfs.core.windows.net`
Thanks!